int32 speed_cmd
